rfd/0026/README.md

```diff
@@ -59,6 +59,7 @@ state: draft
     - [Changes to CloudAPI](#changes-to-cloudapi)
       - [Not exposing NFS volumes' storage VMs via any of the `Machines` endpoints](#not-exposing-nfs-volumes-storage-vms-via-any-of-the-machines-endpoints)
       - [Volume objects representation](#volume-objects-representation)
+      - [New `volumes` parameter to CreateMachine](#new-volumes-parameter-for-createmachine)
```

Comment:
Should we also add an entry for the new `volumes` property of Machines objects returned by various endpoints?